components/list.css

```diff
@@ -0,0 +1,45 @@
+/* adds basic styling */
+@import "./style-guide.css";
+
+/* badge component */
+@import "./badge.css";
+
+/* LIST */
+
+/*-- Spaced List(normal lists) --*/
+/* common spacing */
+.list{
+    margin: var(--margin-xs);
+    padding: var(--padding-s);
+}
+
+.list-item{
+    padding: var(--padding-s);
+}
+
+/*-- Stacked List --*/
+/* simply added some border and shadow to spaced list */
+.stacked-list>.list-item{
+    box-shadow: var(--list-shadow);
```

Comment:
This will not work. Please check once

Reply:
sorry ig the preview link broke, can you check it on this one https://deploy-preview-8--peaceui.netlify.app/list.html

Reply:
I am talking about your box-shadow. Please check: https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

Reply:
I have defined the variable like this `--list-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24); ` , do let me know if the variable name is misleading, will replace it

Reply:
Okay. Makes sense now 👍🏻